### DISCUSSION TITLE
Error : unable to get timezone at origin , try using locals form Brazil

### DISCUSSION THREAD

**Discussion by Eduardokpn** [transitland/transitland#442] [Q&A] [answered]
I'am try to using api with brazilian places but, this error says the timezone is unable, someone to help me, transitland have the gtfs of sptrans. #

https://transit.land/api/v2/routing/otp/plan?fromPlace=-46.5784072,-23.502852&toPlace=-46.596486,-23.519994&date=2024-10-19&time=13:30&arriveBy=false&api_key=wQybh9975d5P36EXmw86nmp2nz8vJIht

**Comment by drewda** [comment, marked as answer]
Thanks for your interest in the Transitland Routing API.

Currently the Routing API is limited to operators in the United States. We plan to expand coverage in the future to more of the feeds within Transitland.

As coverage expands, this documentation will be updated https://www.transit.land/documentation/routing-api/#geographic-coverage
